tensorboard/plugins/interactive_inference/utils/inference_utils.py

```diff
@@ -693,9 +693,9 @@ def generate_image_from_thubnails(thumbnails, thumbnail_dims):
     with tf.compat.v1.Session():
       keys_to_features = {
           image_feature_name:
-              tf.FixedLenFeature((), tf.string, default_value=''),
+              tf.compat.v1.FixedLenFeature((), tf.string, default_value=''),
```

Comment:
Shouldn't this just become `tf.io.FixedLenFeature`?  It's still in TF 2.0, just only under the `io` package.